src/test/java/org/tuberlin/de/server/controller/DownloadJavaControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.tuberlin.de.server.controller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.anyObject;
+import static org.powermock.api.easymock.PowerMock.*;
+
+/**
+ * Unit test for the {@link DownloadJavaController} class that controls that a file
+ * is written into the response
+ */
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created by Fabian on 10.12.2015.
+ */
```

Comment:
Better have only one description per public java class. 
In my opinion, we should omit these "Created by..." lines and instead use something more descriptive, 
as you did a few lines above.